healthcheck/gravitino-healthcheck.sh

```diff
@@ -23,22 +23,24 @@ max_attempts=3
 attempt=0
 success=false
 
+HOST_IP=${GRAVITINO_HOST_IP:-127.0.0.1}
+
 while [ $attempt -lt $max_attempts ]; do
-  response=$(curl -X GET -H "Content-Type: application/json" http://127.0.0.1:8090/api/version)
-  
-  if echo "$response" | grep -q "\"code\":0"; then
-    success=true
-    break
-  else
-    echo "Attempt $((attempt + 1)) failed..."
-    sleep 1
-  fi
-  
-  ((attempt++))
+	response=$(curl -X GET -H "Content-Type: application/json" http://${HOST_IP}:8090/api/version)
+
+	if echo "$response" | grep -q "\"code\":0"; then
+		success=true
+		break
+	else
+		echo "Attempt $((attempt + 1)) failed..."
+		sleep 1
+	fi
+
```

Comment:
Please keep before indent space.
gravitino-playground project doesn't have a shell script style check?

Reply:
Done.

> gravitino-playground project doesn't have a shell script style check?

No, we don't have style check for shell script for now.